Docs: Clarify that experimental feature toggles aren't listed (#128190)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

> This page contains a list of available feature toggles. To learn how to turn on feature toggles, refer to our [Configure Grafana documentation](../#feature_toggles). Feature toggles are also available to Grafana Cloud Advanced customers. If you use Grafana Cloud Advanced, you can open a support ticket and specify the feature toggles and stack for which you want them enabled.
> 
> This page doesn't list experimental feature toggles. To learn whether a feature uses an experimental feature toggle, refer to the documentation for that feature.
> 

Reviewer: How about just:

```suggestion
This page contains a list of available (non-experimental) feature toggles. To learn how to turn on feature toggles, refer to our [Configure Grafana documentation](../#feature_toggles). Feature toggles are also available to Grafana Cloud Advanced customers. If you use Grafana Cloud Advanced, you can open a support ticket and specify the feature toggles and stack for which you want them enabled.
```

Author: Thanks for the suggestion, and for flagging this! I ended up going with a single concise sentence that still points readers to the feature's own docs, which I think captures the same intent. Quick heads-up: index.md is machine-generated from generateDocsMD() in toggles_gen_test.go, so I applied the change in the template and regenerated the page (make gen-feature-toggles) rather than committing the suggestion directly, since committing it on this file alone would fail the generation test. Thanks again for the review!
